devimint/src/tests.rs

```diff
@@ -165,17 +171,13 @@ pub async fn latency_tests(
         LatencyTest::LnSend => {
             info!("Testing latency of ln send");
             let mut ln_sends = Vec::with_capacity(iterations);
-            for i in 0..iterations {
-                let invoice = cln
-                    .invoice(
-                        1_000_000,
-                        format!("Description{i}"),
-                        format!("Label{}", rand::random::<u64>()),
-                    )
-                    .await?;
+            for _ in 0..iterations {
+                let invoice = gw_ldk.create_invoice(1_000_000).await?;
```

Comment:
will this work if you don't have lnv2 module? (in fedi repo we don't)

Reply:
Yes! It is creating a lightning invoice from the gateway's lightning node, not on behalf of a fedimint client. Will work independently of LNv2

Reply:
is this accessible using cli? we have this in fedi 

```rust
    async fn cln_pay_invoice(invoice_string: &str) -> anyhow::Result<()> {
        cmd!(ClnLightningCli, "pay", invoice_string).run().await?;
        Ok(())
    }
```